astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
I'm fine with `AirmassRange`, but I'm strongly against ``AirmassAnything(None,2)``.  Honestly, I'm not sure there's any use case for an "annulus" in Airmass: any real constraints I know of are more reasonably phrased as altitude.  The whole reason for this constraint is mainly a convenience for the (extremely common) case of "my spectra will be junk if they're below airmass 2" or similar.